diff --git a/node_modules/@maticnetwork/maticjs/dist/matic.node.js b/node_modules/@maticnetwork/maticjs/dist/matic.node.js
index 21019b6..96c1ae3 100644
--- a/node_modules/@maticnetwork/maticjs/dist/matic.node.js
+++ b/node_modules/@maticnetwork/maticjs/dist/matic.node.js
@@ -6464,7 +6464,6 @@ if (typeof process === 'undefined' || process.type === 'renderer' || process.bro
  * Module dependencies.
  */
 
-const tty = __webpack_require__(/*! tty */ "tty");
 const util = __webpack_require__(/*! util */ "util");
 
 /**
@@ -6606,14 +6605,8 @@ exports.inspectOpts = Object.keys(process.env).filter(key => {
 	return obj;
 }, {});
 
-/**
- * Is stdout a TTY? Colored output is enabled when `true`.
- */
-
 function useColors() {
-	return 'colors' in exports.inspectOpts ?
-		Boolean(exports.inspectOpts.colors) :
-		tty.isatty(process.stderr.fd);
+	return false
 }
 
 /**
@@ -12064,17 +12057,6 @@ module.exports = require("merkle-patricia-tree");
 
 /***/ }),
 
-/***/ "tty":
-/*!**********************!*\
-  !*** external "tty" ***!
-  \**********************/
-/*! no static exports found */
-/***/ (function(module, exports) {
-
-module.exports = require("tty");
-
-/***/ }),
-
 /***/ "util":
 /*!***********************!*\
   !*** external "util" ***!
